core/src/test/resources/spark-events/local-1425081759268/EVENT_LOG_1

```diff
@@ -0,0 +1,88 @@
+{"Event":"SparkListenerBlockManagerAdded","Block Manager ID":{"Executor ID":"<driver>","Host":"localhost","Port":57967},"Maximum Memory":278302556,"Timestamp":1425081759407}
```

Comment:
I might be missing something, why do we still have test files using the old log format? Shouldn't we just replace all old test logs using the new format?